test/BUILD.gn

```diff
@@ -215,6 +218,7 @@ test("brave_unit_tests") {
     "//components/domain_reliability:domain_reliability",
     "//components/language/core/browser",
     "//components/os_crypt:test_support",
+    "//components/page_load_metrics/common",
```

Comment:
shouldn't we add ```//components/page_load_metrics/common:page_load_metrics_mojom``` here?
```bandwidth_savings_predictor_unittest.cc``` includes ```components/page_load_metrics/common/page_load_metrics.mojom.h```

Reply:
shouldn't we get a gn check error if that was the case?

Reply:
yeah, I was curious why it didn't error. it's actually included via ```//chrome/common```:
```
//brave/test:brave_unit_tests --[public]-->       
//brave/test:brave_test_support_unit --[public]-->
//chrome/common:common --[public]-->
//components/page_load_metrics/common:page_load_metrics_mojom
```